EIPS/eip-1193.md

```diff
@@ -366,7 +376,13 @@ All supported RPC methods **MUST** be identified by unique strings.
 
 Providers **MAY** support whatever RPC methods required to fulfill their purpose, standardized or otherwise.
 
-If a Provider supports a method defined in a finalized EIP, the Provider's implementation of this method **MUST** adhere to the method's specification.
+If a Provider supports a method defined in a finalized EIP, the Provider's implementation of this method **MUST** adhere to the method's specification. This includes:
```

Comment:
This statement feels cyclical?  I read it as "If a provider supports X, then a provider must support X".  Perhaps I'm misunderstanding something?

Reply:
It is a bit cyclical, although I go through the trouble of explicitly defining what we mean by "supported RPC method" above:
> A "supported RPC method" is any RPC method that may be called via the Provider.

If you have any ideas for how to express it in a less cyclical fashion, I'm all ears!

Reply:
My vote is to just remove it, as I don't understand the value it adds to the specification.